dev/release/utils-prepare.sh

```diff
@@ -139,6 +140,12 @@ update_versions() {
   fi
   rm -f NEWS.md.bak
   git add NEWS.md
+
+  find . -name "_pkgdown.yml" -exec sed -i.bak -E -e \
+    "s|(github\\.com/apache/arrow/go)/v[0-9]+|\1/v${major_version}|g" {} \;
+
```

Comment:
Could you remove this empty line because the above `sed` and this `rm` are related?

```suggestion
```